fUrdf:  "SAKURA_RBDL.urdf"
fRobot: "robot_config.xml"
fIk:    "ik_config.xml"
fEnum:  "enum_config.xml"
